open_academy/models/open_academy_course.py

```diff
@@ -9,3 +9,16 @@ class OpenAcademyCourse(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users")
     sessions_ids = fields.One2many("open.academy.session", "course_id")
+
+    _sql_constraints = [
+        ("title_description_diferents", "check(title!=description)",
+        "the title and the description must be diferents"),("title_unique", "unique(title)",
+        "this course title is already taken "),
+    ]
+
+    def copy(self, default=None):
+        if default == None:
+            default = {}
+        default["title"] = self.title + " copy"
```

Comment:
The Exercise says: `Re-implement your own “copy” method which allows to duplicate the Course object, changing the original name into “Copy of [original name]”.`

```suggestion
        default["title"] = "Copy of %s" % self.title
```